chain/x/oracle/keeper/keeper.go

```diff
@@ -153,3 +156,13 @@ func (k Keeper) GetNextOracleScriptID(ctx sdk.Context) types.OracleScriptID {
 	store.Set(types.OracleScriptCountStoreKey, bz)
 	return types.OracleScriptID(oracleScriptCount + 1)
 }
+
+// TODO: Hack for use
```

Comment:
// BindPort defines a wrapper function for the ort Keeper's function in
// order to expose it to module's InitGenesis function